### PR TITLE
PLT-2187 Add MFA functionality

### PR DESCRIPTION
**Pull request by jwilander**
There is an enterprise PR to go along with this.

### CODE REVIEW

File: mattermost.go [hunk 1]
```diff
@@ -29,7 +29,9 @@ import (
 	_ "github.com/mattermost/platform/model/gitlab"
 
 	// Enterprise Deps
+	_ "github.com/dgryski/dgoogauth"
 	_ "github.com/go-ldap/ldap"
+	_ "github.com/jwilander/rsc/qr"
```
Comment (coreyhulen) [inline mattermost.go]: should we move this to mattermost?

Comment (jwilander) [inline mattermost.go]: Yeah, if you can make a repo called rsc and give me write access I can move it over

**Comment by coreyhulen** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**